packages/dataviews/src/components/dataviews-filters/style.scss

```diff
@@ -375,3 +376,11 @@
 .dataviews-search {
 	width: fit-content;
 }
+
+.dataviews-filters__user-input-widget {
+	padding: $grid-unit-20;
+
+	.components-input-control__prefix {
+		padding-left: $grid-unit-10;
+	}
+}
```

Comment:
Needs an empty line

Reply:
Fixed in 53f987cfb452c62ef4f87f41816e5df25fa58b21.